scripts/download_chain.sh

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/bash
+
+set -euxo pipefail
+
+sub2sql --chain=wss://889e30bc.testnet.fractalprotocol.com \
```

Comment:
Unsure if ready yet, but wouldn't `nodes.testnet.fractalprotocol.com` be a better host to connect to?

Reply:
Done.